src/main/java/se/sundsvall/supportmanagement/service/scheduler/emailreader/EmailReaderWorker.java

```diff
@@ -50,9 +48,8 @@ public EmailReaderWorker(final EmailReaderProperties emailReaderProperties, fina
 	}
 
 
-	@Scheduled(initialDelayString = "${scheduler.emailreader.initialDelay}", fixedRateString = "${scheduler.emailreader.fixedRate}", timeUnit = TimeUnit.SECONDS)
-	@SchedulerLock(name = "fetch_emails", lockAtMostFor = "${scheduler.emailreader.shedlock-lock-at-most-for}")
-	void getAndProcessEmails() {
+	@Transactional
```

Comment:
I guess you don't need the `@Transactional` here, since it's defined on class level?